GoodreadsEDA/all_bks.py

```diff
@@ -114,7 +97,6 @@ def create_even_samples(df, df2):
 df_after_3mo = df_after[df_after.index <= df_after["3mo_after_release"]]
 
 # Balance the data:
-
```

Comment:
All this repetition with minor deviations is a good spot for implementing a centralized method to take care of those functionalities

Reply:
I'll add it to the to do! That last script was more of less for getting the output for my writeup, and as such has a lot to be desired. I'm planning on working through it today since it's a snowday up where I am.